### DISCUSSION TITLE
Erro ao validar xml

### DISCUSSION THREAD

**Discussion by sidneiJr1984** [qualitaocupacional/libesocial#12] [General]
[S2220_1.zip](https://github.com/qualitaocupacional/libesocial/files/14024787/S2220_1.zip)
Bom dia,

Estou tendo alguns erros na validação do XML, mesmo pegando um de teste do repositorio o erro de validação ainda ocorre. 
![erro_1](https://github.com/qualitaocupacional/libesocial/assets/45664938/b338b775-bfa8-4ef0-a44e-03296406156b)

Sera que vocês poderiam me auxiliar?

**Comment by andidevel** [comment]
Olá @sidneiJr1984,

Não sei qual é o erro que está apresentando para o seu `XML`, mas o exemplo que você mostrou, usando o `XML` de testes do repositório está correto. Esse `XML` não tem assinatura digital, portanto é um `XML` inválido para ser enviado para o **eSocial**. Você pode consultar a pasta **tests** para dar uma olhada nos testes unitários.

Se você quiser saber quais são os erros de validação de um `XML` de evento, no [README](https://github.com/qualitaocupacional/libesocial/blob/main/README.md) do projeto tem alguns exemplos. Procure por "**Validando um evento**".

Também seria interessante usar o código da branch **develop**. Tem umas atualizações importantes lá, mas ainda não está pronto para fazer merge na **main**. Teste bastante antes de usar em produção se a versão que está na **main** não lhe atender.


Espero ter ajudado.
